src/routes/download/__layout.svelte

```diff
@@ -0,0 +1,66 @@
+<script lang="ts">
+	import { Button, ProgressRing } from "$lib";
+</script>
+
+<slot/>
+
+<div class="download-page">
+	<h1>Downloading</h1>
+	<ProgressRing size={64}/>
+	<p>Starting download of sideload package...</p>
```

Comment:
Should this be a thank you message instead? The app installer handless the downloading part.

Reply:
Sounds good.